### DISCUSSION TITLE
post/ipv6-differences/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#21] [Announcements]
# post/ipv6-differences/

如题, 已有一些 IPv4 的基础知识, 来速成一下 IPv6… 一直没仔细研究过 IPv6 的具体规范, 属于是在使用过程中自行总结加搜索发现了一些规律… 直到遇到了南京大学的无力吐槽校园网配置, 正好来基于 IPv4 来速成 IPv6 (记录 v6 比 v4 新增/不同的部分). 为什么需要 IPv6 及 IPv6 现况 网际协议第6版（英语：Internet Protocol ve

https://blog.lyc8503.net/post/ipv6-differences/

**Comment by tongkl1** [comment]
Update: 我在胡说八道，已更正

**Comment by GLASS20** [comment]
"Wikipedia 说 IPv4 目标是取代 IPv6 网络" 写错了喵

**Reply by lyc8503** [reply]
感谢提醒，已更正😂

**Comment by chengnan049** [comment]
https://hello.fe80.cn/pages/ipv6/i6.html 地址已失效
